build.sbt

```diff
@@ -294,8 +306,11 @@ lazy val compilerInterface = (project in internalPath / "compiler-interface")
 // Compiler-side interface to compiler that is compiled against the compiler being used either in advance or on the fly.
 //   Includes API and Analyzer phases that extract source API and relationships.
 lazy val compilerBridge: Project = (project in internalPath / "compiler-bridge")
-  .dependsOn(compilerInterface % "compile;test->test",
-             /*launchProj % "test->test",*/ zincApiInfo % "test->test")
+  .dependsOn(
+    compilerInterface % "compile;test->test",
+//  launchProj % "test->test",
```

Comment:
I think that the time to remove this comment has come.
https://github.com/sbt/sbt-zero-thirteen/blob/0.13/build.sbt#L380